index.html

```diff
@@ -4975,6 +4975,646 @@ <h2 id="AudioWorkerParamDescriptor">
+            script runs. It is designed to enable the generation, processing,
+            and analysis of audio data directly using JavaScript in the audio
+            <a>rendering thread</a>. The user-supplied script code is evaluated
+            in this scope to define an
+            <a><code>AudioWorkletProcessor</code></a>.
+          </p>
+          <dl title="interface AudioWorkletGlobalScope : WorkletGlobalScope"
+          class="idl">
+            <dt>
+              void registerAudioWorkletProcessor (DOMString name, Function
+              classConstructor)
+            </dt>
+            <dd>
+              <p>
+                Registers a definition of a class that is derived from
+                <a>AudioWorkletProcessor</a>.
+              </p>
+              <dl class="parameters">
+                <dt>
+                  DOMString name
+                </dt>
+                <dd>
+                  A string key that represents a class definition to be
+                  registered. This key is used to resolve an
+                  <a><code>AudioWorkletProcessor</code></a> during construction
+                  of an <a><code>AudioWorkletNode</code></a>. Throws an
+                  exception if another
+                  <a><code>AudioWorkletProcessor</code></a> has already
+                  registered this key.
+                </dd>
+                <dt>
+                  Function classConstructor
+                </dt>
+                <dd>
+                  A class definition extended from
+                  <a>AudioWorkletProcessor</a>. Throws if this is not a class
+                  definition or it does not extend
+                  <a><code>AudioWorkletProcessor</code></a>.
+                </dd>
+              </dl>
+            </dd>
+          </dl>
+        </section>
+        <section>
+          <h2 id="AudioWorkletProcessor">
+            The <dfn>AudioWorkletProcessor</dfn> interface
+          </h2>
+          <p>
+            This interface represents an audio processing module that runs on
+            the audio <a>rendering thread</a>. It is defined in an
+            <a><code>AudioWorkletGlobalScope</code></a> and the definition of
+            the class manifests the actual audio processing mechanism of a
+            custom audio node. <a><code>AudioWorkletProcessor</code></a> can
+            only be instantiated by the construction of an
+            <a><code>AudioWorkletNode</code></a> instance.
+          </p>
+          <dl title="interface AudioWorkletProcessor" class="idl">
+            <dt>
+              readonly attribute static sequence&lt;AudioParamDescriptor&gt;
+              parameterDescriptors
+            </dt>
+            <dd>
+              <p>
+                This getter returns an array containing
+                <a>AudioParamDescriptor</a>s, which are descriptors for each
+                controllable parameter on an <a>AudioNode</a>. This static
```

Comment:
Does static have a meaning in the Web Platform, or is it just a C++ thing here ?